src/block-management/block-holder.js

```diff
@@ -170,10 +171,12 @@ export class BlockHolder extends React.Component<PropsType, StateType> {
 		const borderColor = isSelected ? focusedBorderColor : 'transparent';
 
 		return (
-			<TouchableWithoutFeedback onPress={ this.onFocus } >
+			<TouchableWithoutFeedback accessible={ false } accessibilityLabel="block-container" onPress={ this.onFocus } >
```

Comment:
`accessible={ false }` is probably not intuitive enough of why its `false`. Can you share related info/insights? Thanks!

Reply:
Right, I can add an inline comment here to help. The reason for this comes from the [RN docs on accessibility](https://facebook.github.io/react-native/docs/accessibility) 

> accessible (iOS, Android)
When true, indicates that the view is an accessibility element. When a view is an accessibility element, it groups its children into a single selectable component. By default, all touchable elements are accessible.

In this case we'll need to be able to access the children so accessible is set to false in this case.